index.html

```diff
@@ -0,0 +1,78 @@
+---
+layout: default
+---
+
+<header class="ui-menu">
+  <img src="/static/img/ceclogo.svg" class="ui-menu__logo">
+  <nav class="ui-nav">
+    <ul>
+      <li><a href="/" rel="index">Home</a></li>
+      <li><a href="#about">About CEC</a></li>
+      <li><a href="#categories">Categories</a></li>
+      <li><a href="#schedule">Schedule</a></li>
+    </ul>
+  </nav>
+</header>
+<section class="ui-section" id="home">
+  <div class="ui-section__item">
+    {% include nameplate.html %}
+  </div>
+</section>
+<ul class="stat-banner">
+  {% for stat in site.data.stats %}
+  <li class="stat-banner__block">
+    <span class="stat-banner__number">{{ stat.number }}</span>
+    <span class="stat-banner__label">{{ stat.label }}</span>
+  </li>
+  {% endfor %}
+</ul>
+<section class="ui-section" id="about">
+  <div class="ui-section__item">
+    <h1>
+      <span class="type-headline">Canada's premier engineering competition is back,</span>
+      <span class="type-headline">hosted by the nation's most innovative university.</span>
+    </h1>
+    <p>The Canadian Engineering Competition (CEC) is Canada's premier competition for undergraduate engineering students. Since its inception in 1985, CEC has invited the top 150 engineering students each year to compete in one of eight exciting competition categories.</p>
+    <p>In order to qualify, students must first win in an intra-university competition, and then achieve top place in their respective <a href="">Western</a>, <a href="">Ontario</a>, <a href="">Québec</a>, and <a href="">Atlantic</a> regional competitions. In 2019, the culmination of those efforts will bring competitors to showcase their skills at the University of Waterloo in the heart of our nation's tech and innovation hub.</p>
```

Comment:
the links here are empty, but this can be in a separate PR

Reply:
yeah, i just haven't grabbed the official links for those competition regions yet whoops. will do so in another PR (will edit this comment later to link to it).